### PR TITLE
Create first draft of suspect commit documentation

### PR DESCRIPTION
**Pull request by jhgilbert**
### What does this PR do?
Adds documentation on Error Tracking > Suspect Commits.

### Motivation
DOCS-6806

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by jhgilbert** [comment]
Hi @AyushK1, interested in your thoughts on this draft of Suspect Commits documentation. I kept it simple, so let me know if you want more detail anywhere. When you're happy with it, I'll also add it to the other Error Tracking sections of the docs.

🔴 Also want to flag that I guessed at which GitHub integration permissions were needed, since the Confluence doc just said "correct permissions". If more permissions are needed, I can add those.

File: content/en/logs/error_tracking/suspect_commits.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+title: Suspect Commits
+kind: documentation
+disable_toc: false
+---
+## Overview
+
+For some issues, Error Tracking can identify a suspect commit, helping you determine the root cause and resolve the issue. This feature is automatically enabled when its [setup requirements](#setup) are met.
```
Comment (meghanlo) [inline content/en/logs/error_tracking/suspect_commits.md:8]:
```suggestion
Error Tracking can identify a suspect commits, helping you pinpoint the root cause of your errors and resolve issue faster. This feature is automatically enabled on issues when the [setup requirements](#setup) are met.
```

File: content/en/logs/error_tracking/suspect_commits.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+{{< img src="logs/error_tracking/suspect_commit.png" alt="A suspect commit as it is displayed in the Datadog UI" style="width:100%" >}}
+
+If a suspect commit is associated with an issue, it is displayed on the issue panel, as shown in the highlighted area of the image below.
```
Comment (meghanlo) [inline content/en/logs/error_tracking/suspect_commits.md:12]:
```suggestion
Once a suspect commit has been identified, it is displayed on the issue panel as shown in the highlighted area of the image below.
```

**Review by meghanlo — COMMENTED** [review]
a couple of nits

**Review by aliciascott — CHANGES_REQUESTED** [review]
Looks good just a few small suggestions! Let me know if you need a re-review :)

**Review by meghanlo — APPROVED** [review]

**Review by aliciascott — APPROVED** [review]
looks good!

**Pull request merged**